archive/archive_asset_lookup/src/endpoint.py

```diff
@@ -0,0 +1,103 @@
+import os
+import tempfile
+import logging
+import boto3
+import botocore
+import settings
+from boto3.dynamodb.conditions import Key
+
+
+def lambda_handler(event, context):
+
+    # setup logging and AWS clients
+    logger = logging.getLogger()
+    dynamodb = boto3.resource('dynamodb', region_name=settings.AWS_REGION)
+    s3 = boto3.resource('s3', region_name=settings.AWS_REGION)
+
+    logger.info('got event{}'.format(event))
```

Comment:
We usually try and break the entry point from the main application code and hand dependencies to that for easier testing.

e.g.

```python
def lambda_handler(event, context):
  logger = logging.getLogger()
  dynamodb = boto3.resource('dynamodb', region_name=settings.AWS_REGION)
  s3 = boto3.resource('s3', region_name=settings.AWS_REGION)

def asset_lookup(dynamodb, table_name, identifier):
  table = dynamodb.Table(table_name)
  # ... etc etc 
```

Reply:
Yep good plan, I'll work that in for next version.